components/src/core/Drawer/DrawerHeader.tsx

```diff
@@ -1,25 +1,36 @@
 import React, { ReactNode, useCallback } from 'react';
 import PropTypes from 'prop-types';
-import { Theme } from '@mui/material/styles';
-import createStyles from '@mui/styles/createStyles';
-import makeStyles from '@mui/styles/makeStyles';
 import Toolbar, { ToolbarProps } from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import Divider from '@mui/material/Divider';
 import IconButton from '@mui/material/IconButton';
 import { useDrawerContext } from './DrawerContext';
+import drawerHeaderClasses, {
+    DrawerHeaderClasses,
+    DrawerHeaderClassKey,
+    getDrawerHeaderUtilityClass,
+} from './DrawerHeaderClasses';
+import { cx } from '@emotion/css';
 import clsx from 'clsx';
+import { styled } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
 
-type DrawerHeaderClasses = {
-    root?: string;
-    background?: string;
-    content?: string;
-    navigation?: string;
-    nonCLickable?: string;
-    nonClickableIcon?: string;
-    railIcon?: string;
-    subtitle?: string;
-    title?: string;
+const useUtilityClasses = (ownerState: DrawerHeaderProps): Record<DrawerHeaderClassKey, string> => {
+    const { classes } = ownerState;
+    const slots = {
+        root: ['root'],
+        background: ['background'],
+        content: ['content'],
+        navigation: ['navigation'],
+        nonCLickable: ['nonCLickable'],
```

Comment:
looks like this has been a long running typo, but can we fix this
```suggestion
        nonClickable: ['nonClickable'],
```